rss_parser/models/atom/feed.py

```diff
@@ -0,0 +1,61 @@
+from typing import Optional
+
+from rss_parser.models import XMLBaseModel
+from rss_parser.models.atom.entry import Entry
+from rss_parser.models.image import Image
+from rss_parser.models.types.date import DateTimeOrStr
+from rss_parser.models.types.only_list import OnlyList
+from rss_parser.models.types.tag import Tag
+from rss_parser.pydantic_proxy import import_v1_pydantic
+
+pydantic = import_v1_pydantic()
+
+
+class RequiredAtomFeedMixin(XMLBaseModel):
+    feed_id: Tag[str] = pydantic.Field(alias="id")
+    "Identifies the feed using a universally unique and permanent URI."
+
+    title: Tag[str]
+    "Contains a human readable title for the feed."
+
+    updated: Tag[DateTimeOrStr]
+    "Indicates the last time the feed was modified in a significant way."
+
+
+class RecommendedAtomFeedMixin(XMLBaseModel):
+    author: Optional[Tag[str]] = None
```

Comment:
This is think should be another model, since in w3 I see a schema like this
```xml
<author>
  <name>John Doe</name>
  <email>JohnDoe@example.com</email>
  <uri>http://example.com/~johndoe</uri>
</author>
```

Also this seems like a good candidate for `OnlyList`

>  A feed may have multiple author elements. A feed must contain at least one author element unless all of the entry elements contain at least one author element.

Reply:
Same for `contributor`

```xml
<contributor>
  <name>Jane Doe</name>
</contributor>
```